packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -111,79 +89,87 @@ export class ProjectService {
         return project;
     }
 
-    async create(user: SessionUser, data: CreateProject): Promise<Project> {
+    async create(
```

Comment:
create endpoint will:

1. create a job
2. return the job id
3. continue remainder of work in the background